src/main/kotlin/com/exactpro/th2/codec/oracle/logminer/LogMinerTransformer.kt

```diff
@@ -156,6 +569,15 @@ class LogMinerTransformer(private val config: LogMinerConfiguration) : IPipeline
             LOG_MINER_TABLE_NAME_COLUMN,
         )
 
+        internal fun truncateFromWhereClause(query: String): String {
+            val whereIndex = query.indexOf(WHERE_CLAUSE, ignoreCase = true)
```

Comment:
Just to point out this might not work as you expect. If we have a select statement like this `SELECT City as WhereWeAre FROM Locations WHERE Country != Russia` you will get the wrong result

Reply:
Currently this code should be able to parse INSERT/UPDATE (without where) statements. We will add special test for other option if needed